xml/System/Type.xml

```diff
@@ -6135,12 +6135,20 @@ One of the elements in the <paramref name="types" /> array is <see langword="nul
         <Parameter Name="types" Type="System.Type[]" Index="3" FrameworkAlternate="net-9.0" />
       </Parameters>
       <Docs>
-        <param name="name">To be added.</param>
-        <param name="genericParameterCount">To be added.</param>
-        <param name="bindingAttr">To be added.</param>
-        <param name="types">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="name">The string containing the name of the method to get.</param>
+        <param name="genericParameterCount">The number of generic type parameters of the method.</param>
+        <param name="bindingAttr">A bitwise combination of the enumeration values that specify how the search is conducted.
+
+ -or-
+
+ <see cref="F:System.Reflection.BindingFlags.Default" /> to return <see langword="null" />.</param>
+        <param name="types">An array of <see cref="T:System.Type" /> objects representing the number, order, and type of the parameters for the method to get.
+
+-or-
+
+An empty array of <see cref="T:System.Type" /> objects (as provided by the <see cref="F:System.Type.EmptyTypes" /> field) to get a method that takes no parameters.</param>
+        <summary>Searches for the specified method whose parameters match the specified generic parameter count, argument types, using the specified binding constraints.</summary>
+        <returns>An object representing the method that matches the specified generic parameter count, argument types and binding constraints, if found; otherwise, <see langword="null" />.</returns>
```

Comment:
```suggestion
        <returns>An object representing the method that matches the specified generic parameter count, argument types, and binding constraints, if found; otherwise, <see langword="null" />.</returns>
```